OptimizelySDKCore/OptimizelySDKCore/OPTLYProjectConfig.m

```diff
@@ -362,6 +369,11 @@ - (BOOL)setForcedVariation:(nonnull NSString *)experimentKey
 
 #pragma mark -- Property Getters --
 
+/// Array representing supported datafile versions.
++ (NSArray *)supportedDatafileVersions {
```

Comment:
Why function is needed, it will initialize array again and again. Make it static and allocate values in array.